addons/website_quote/views/website_quotation.xml

```diff
@@ -148,7 +151,7 @@
                                   </a>
                               </div>
                           </div>
-                          <div class="text-center" t-if="need_payment" style="padding: 10px">
+                          <div class="text-center" t-if="need_payment or (quotation.state in ('sale') and quotation.require_payment and quotation.invoice_status != 'invoiced' and tx_state != 'done')" style="padding: 10px">
```

Comment:
The goal is to be able to display the button 'Pay' if the order has been validated in frontend, so if the invoice is already created too. 

If there is an upsell, the button pay will not be displayed as the tx is already done...